PiwikTracker.php

```diff
@@ -1070,11 +1107,17 @@ public function getUrlTrackAction($actionUrl, $actionType)
      *
      * Allowed only for Super User, must be used along with setTokenAuth()
      * @see setTokenAuth()
-     * @param string $dateTime Date with the format 'Y-m-d H:i:s', or a UNIX timestamp
+     * @param string|DateTimeInterface $dateTime Date with the format 'Y-m-d H:i:s', or a UNIX timestamp, or DateTimeInterface object
+     * @return $this
      */
     public function setForceVisitDateTime($dateTime)
     {
-        $this->forcedDatetime = $dateTime;
+        if ($dateTime instanceof DateTimeInterface) {
```

Comment:
Let's maybe remove it for now and when working on a new version of PHP Tracker we could use DateTimeInterface by default.